nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/util/SFTPTransfer.java

```diff
@@ -236,6 +238,12 @@ public int select(final LsEntry entry) {
                         return LsEntrySelector.CONTINUE;
                     }
 
+                    // if is a link and we're supposed to follow symlink
+                    if (symlink && entry.getAttrs().isLink()) {
+                        links.add(entry);
+                        return LsEntrySelector.CONTINUE;
+                    }
+
```

Comment:
Same as before, wondering if it'd be better to avoid code duplication:
````java
                    if ( (recurse && entry.getAttrs().isDir()) || (symlink && entry.getAttrs().isLink()) ) {
                         subDirs.add(entry);
                        return LsEntrySelector.CONTINUE;
                    }
````